js/nav_bar.js

```diff
@@ -0,0 +1,19 @@
+const navigationButton = document.querySelector(".nav_button");
+const navMenuContainer = document.querySelector(".nav_menu_container");
+const navMenuRecipes = document.querySelector(".nav_menu_recipes");
+
+let open = false;
+
+navigationButton.addEventListener("click", toggleNavBar);
+
+function toggleNavBar() {
+  event.preventDefault();
+  open = !open;
+  if (open) {
+    navMenuContainer.style.display = "block";
+    navMenuRecipes.style.display = "flex";
+  } else {
+    navMenuContainer.style.display = "none";
+    navMenuRecipes.style.display = "none";
+  }
```

Comment:
// should we also add ...style.overflow = "hidden" here?

Reply:
Ne, das brauchen wir nicht, weil es sowieso nicht gescrollt wird.